tests/UI/pages/BO/customerService/customerService/view.js

```diff
@@ -0,0 +1,51 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+class ViewCustomer extends BOBasePage {
+  constructor() {
+    super();
+
+    this.pageTitle = 'Customer Service > View •';
+
+    // Selectors
+    this.threadBadge = 'span.badge';
+    this.messageDiv = '#content div.message-item-initial';
+    this.yourAnswerForm = '#content div.row div:nth-child(5) form';
+    this.ordersAndMessagesForm = '#content div.row div:nth-child(6)';
```

Comment:
No better selectors here :disappointed:  ?

Reply:
no :/

Reply:
Not worth to add some? We shouldn't hesitate on this I think

Reply:
Boubker regularly go through all the weird selectors and change them directly in the .tpl files like a hero 🦸

Reply:
It's a little harder to add selectors in legacy pages.